common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -953,7 +954,8 @@ def make_course_usage_key(self, course_key):
         Return a valid :class:`~opaque_keys.edx.keys.UsageKey` for this modulestore
         that matches the supplied course_key.
         """
-        return BlockUsageLocator(course_key, 'course', 'course')
+        locator_cls = CCXBlockUsageLocator if isinstance(course_key, CCXLocator) else BlockUsageLocator
```

Comment:
I've done a really poor job about documenting/telling folks about CCX. I've got a presentation prepared on the topic. Maybe lunch material on a slow week.